.github/workflows/test.yml

```diff
@@ -37,6 +37,22 @@ jobs:
           flake8-version: 6.0.0
 
   # JOBS MUST START WITH test !!!!
+  test_notebooks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Set up Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          pip install -U pytest pytest-xdist nbmake
+          pip install -e ".[full]"
+      - name: Run notebooks
+        run: pytest --nbmake -n=auto --ignore=./tutorials/japanese/ --ignore=./tutorials/chinese/ --ignore=./tutorials/greek/ --ignore=./tutorials/portuguese/ ./tutorials/
```

Comment:
@Yard1 @ngupta23 Any idea how to exclude subfolders with one ignore or directly in the path? with some wildcard magic perhaps...

Reply:
Can you delete the subfolders in the installation steps (like we do for some of the unit tests)?